kratos/gui/screen.py

```diff
@@ -0,0 +1,53 @@
+"""
+The base class for creating more complex GUI screens
+"""
+
+import os
+
+import PyQt5.QtCore as QtCore
+import PyQt5.QtGui as QtGui
+import PyQt5.QtQuick as QtQuick
+
+
+class Screen:
+    """
+    Base screen class that creates an Qt application with argument passing and QML
+    resource loading
+    """
+
+    # pylint: disable=too-few-public-methods
+    # This is a base class and has (at the moment) only limited functionality
+
+    def __init__(self, argv=[]):
```

Comment:
Mutable default arguments should be avoided: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

```Python
    def __init__(self, argv=None):
        """
        Ctor
        :param argv: Arguments passed to the underlying QGuiApplication
        """
        argv = argv or []
```
In the above snippet argv is argv if it's evaluated as True and [] otherwise. None evaluates as False in if-statements.